src/compas_plotters/meshplotter.py

```diff
@@ -153,19 +166,19 @@ def draw_vertices(self,
         ----------
         keys : list
             The keys of the vertices to plot.
-        radius : list
+        radius : {list, dict}
             A list of radii for the vertices.
-        text : list
+        text : {list, dict}
             Strings to be displayed on the vertices.
-        facecolor : list
+        facecolor : {list, dict}
             Color for the vertex circle fill.
-        edgecolor : list
+        edgecolor : {list, dict}
             Color for the vertex circle edge.
-        edgewidth : list
+        edgewidth : {list, dict}
             Width for the vertex circle edge.
-        textcolor : list
+        textcolor : {list, dict}
             Color for the text to be displayed on the vertices.
-        fontsize : list
+        fontsize : {list, dict}
             Font size for the text to be displayed on the vertices.
```

Comment:
the docs are indeed wrong. however, the values can actually also be single values, and in the case of `text` you can even use the keywords "key" or "index". so perhaps it should be
```python
radius : {float, list, dict}
text : {{'index', 'key'}, str, list, dict}
facecolor : {color, list, dict}
edgecolor : {color, list, dict}
edgewidth : {float, list, dict}
textcolor : {color, list, dict}
fontsize : {int, list, color}
```

Reply:
Ok, I was not sure of that. 
It was difficult to me to see all the possible cases.

Reply:
yes i know. the plotters are due for a thorough update :)

Reply:
actually my comment was not entirely correct. it should be
```python
radius : {float, dict}
text : {{'index', 'key'}, str, dict}
facecolor : {color, dict}
edgecolor : {color, dict}
edgewidth : {float, dict}
textcolor : {color, dict}
fontsize : {int, dict}
```